coffee-module/coffee-module-etcd/src/main/java/hu/icellmobilsoft/coffee/module/etcd/producer/CachedEtcdConfigSource.java

```diff
@@ -34,9 +35,10 @@
  */
 public class CachedEtcdConfigSource extends DefaultEtcdConfigSource {
 
+    static boolean active = false;
```

Comment:
A feladat leírásban, azzal hogy "active flag is needed with an extension of `CachedEtcdConfigSource`", arra utaltam, hogy ezt a runtime active flag kezelést egy leszármazottba kellene tenni, hogy aki ezt a `CachedEtcdConfigSource`-ot használja, azt ne befolyásolja. Így, hogy itt van default active = false-al, így rá van minden használó kényzerítve, hogy futásidőben is aktiválja a config source-t, de ez nem minden esetben kell, hogy így működjön, csak ha külön arra van igény.

Reply:
Elnézést, félre értettem a feladatot.